### ISSUE TITLE
[boschshc] Binding title mistyped prevents finding it easily

### ISSUE THREAD

**Issue by mickymix** [openhab/openhab-addons#15607]
I searched for any binding of my latest toy, a Bosch Smart Home 360 camera and...

## Current Behavior
I was able to find the documentation on OpenHab resources but wasn't able to find the binding itself...
Well, the binding is present in the repository, but searching with the brand name "Bosch" did not go through...

So I started worrying not being available for my build... (v4.0.3)

I noticed that the name of the binding is mistyped as "Bosh", instead of "Bosch".

## Possible Solution
If there is no special reason for it (and I did not found any, even after searching for it), I would suggest to fix it. ;-)

## Steps to Reproduce (for Bugs)
1. Open Binding Search Page and type "Bosch"...
2. No "Bosch Smart Home" binding is found.
3. Now retype "Bosh" and the binding is there.

## My Environment
* Version used: OpenHab v4.0.3
* Environment name and version (e.g. Chrome 76, Java 8, Node.js 12.9, ...): Chrome 116.0.5845.188
* Operating System and version (desktop or mobile, Windows 10, Raspbian Buster, ...): Docker Container on QNAP Container Station

**Comment by lsiepel** [comment]
Äs it is not fully clear (to me) what search box you are talking about:

For openHAB.org:
![image](https://github.com/openhab/openhab-addons/assets/13950897/bc50321f-cb5b-422d-99e4-b7739e58e0f6)

For openHAB 4.0.3:
![image](https://github.com/openhab/openhab-addons/assets/13950897/347a47a8-9980-49e6-88d1-2fc2a20147e2)

Maybe you mean some other place, but to me both look fine.

**Comment by mickymix** [comment]
I was talking of the second image: the search which is available within OpenHab itself (and I was referring to the v4.0.3 too).

And you are right: now I searched it once again and I was able to find it.

But since I'm sure of what I saw, I remember having applied one modification only: I have removed the binding, which was already installed (my Bosch Smart Camera 360 does not work with a Bosch Controller, but with its app only, so the binding is not useful for me).

To be able to reproduce the bug, I installed the binding once again and the "search unavailability" only happens when the binding is already installed (as it is mystyped).

The problem (the bug) is that the binding needs to be installed, to be "unsearchable".

I hope having explained myself a little bit more.

_I don't know if I need to edit the main message to report this further discovery or what... I'm not so used to use github... :-)_

**Comment by lsiepel** [comment]
I searcehd the full repo, the only occurence i could find of "bosh" was in the italian translation. I tried to update that from crowdin, but never done that before, so not sure if it was right.

I still wonder why it is different between searched and installed. Maybe @florian-h05 knows where this string comes from?

**Comment by mickymix** [comment]
Thank you for your kind and helpful investigation.

I probably forgot to mention I'm Italian and I use OpenHab in italian too... (for the reasons explained below...)

I did not consider this could be a problem of translation (and effectively, what you found makes sense too).

Being a problem of the Italian translation only, at this point, I'm happy this did not involved many other users and I regret having opened a request for this small glitch only.

I'll have a look at crowdin myself too and validate your fix, if it might help.

Since I always try to avoid translation problems, I try to keep cores in english, but this installation is for "family use" and they strongly prefer local language. :-P

Thank you !

**Comment by lsiepel** [comment]
Hopefully you can fix it in crowdin if i didn't succeed. Atleast we know where the issue is.

**Comment by florian-h05** [comment]
> I still wonder why it is different between searched and installed. Maybe @florian-h05 knows where this string comes from?

This is just guessing, I don’t know it for sure: I guess when the binding is not installed, the name is taken from feature.xml (which is an OSGi/Apache Karaf file). As soon at is gets installed, addon.xml and the translations become available and will be used. 
Also see https://github.com/openhab/openhab-core/issues/1701#issuecomment-704881334, which I just found - I was looking for another explanation, but this one is even better.


> I searcehd the full repo, the only occurence i could find of "bosh" was in the italian translation. I tried to update that from crowdin, but never done that before, so not sure if it was right.

I have just checked CrowdIn, a proof reader approved your translation, so it should get synced to the repo soon. 

FYI for German translations, I have the right to proof read, but only for German.
